python/marvin/contrib/vacs/firefly.py

```diff
@@ -169,10 +200,12 @@ def _make_map(self, parameter=None):
             params)
 
         # get the required arrays
-        binid = self._get_data(ext='SPATIAL_BINID')[self._idx].reshape(
-            76, 76)
+        binid = self._get_data(ext='SPATIAL_BINID')[self._idx].squeeze(axis=0)
         bin1d = self._get_data(ext='SPATIAL_INFO')[self._idx, :, 0][0]
-        prop = self._get_data(ext=parameter + '_voronoi')[self._idx, :, 0][0]
+        try:
+            prop = self._get_data(ext=parameter + '_voronoi')[self._idx, :, 0][0]
+        except:
+            prop = self._get_data(ext=parameter + '_voronoi')[self._idx, :][0]
```

Comment:
Do these code changes, along with the `e(b-v)` -> `e(b_v)` change, also work for Firefly data from DR15, etc?  This should still work for older release data even though we encourage users to use the latest data release.

Reply:
Ok great.  Glad this fixes both releases.  Ideally it would be great if the `signal_noise` plot would work for both releases.  Since we can't fix the DR15 VAC itself, I think it makes sense to try to apply a fix in the code itself.  Do you think that's an easy fix to implement?  If it's too complicated or not worth it since there's DR17, then we might want to log a warning message for the user so they understand that the blank plot is expected.

Reply:
The release version is not a parameter that is known by the methods in the Firefly target class, is it? Is there an easy way to get that so we can add something like "if release!='DR17': ..."?

Reply:
It's not readily available inside the `VACTarget` class.  I think you'd have to pass it into `FireflyTarget` when it's called from within `get_target`.  `get_target` has access to the release through `parent_object.release`.

Reply:
Thanks! That should be fixed now.